lib/resty/core/base.lua

```diff
@@ -129,7 +129,7 @@ end
 local c_buf_type = ffi.typeof("char[?]")
 
 
-local _M = new_tab(0, 17)
+local _M = new_tab(0, 19)
```

Comment:
It seems like `18` is the appropriate size here (`_M.get_request` can only be assigned once). This also appears to be proven by [this test](https://github.com/openresty/lua-resty-core/pull/206/files#diff-cf27e9ecb184dab30525a4fbeb4b119bR69).